/**:
  ros__parameters:

    # 坐标系参数
    base_frame: "base_link"
    gimbal_frame: "livox_frame"
    
    # 话题配置
    terrain_map_topic: "/terrain_map" # 地形图话题
    gimbal_cmd_topic: "/gimbalcontrol"   # 云台控制话题
    tracker_topic: "/tracker/send"       # 自瞄话题
    armors_topic: "/detector/armors"     # 装甲板检测话题
    chassis_topic: "/Chassis"            # 底盘话题
    
    # 障碍物检测参数
    min_bounding_box: 0.5  # 最小障碍物尺寸
    max_bounding_box: 1.5  # 最大障碍物尺寸
    min_scan_points: 20    # 最小扫描点数
    
    # 云台运动参数
    spin_yaw_speed: 2.0    # 云台水平旋转速度
    spin_pitch_speed: 2.0  # 云台俯仰速度
    scan_yaw_angle: 0.05   # 扫描时的yaw角度范围
    max_pitch: 0.30        # 最大俯仰角度
    min_pitch: -0.21       # 最小俯仰角度
    max_yaw_speed: 2.0     # 最大yaw旋转速度
    pitch_more_speed: 0.5  # 仰时yaw速度
    pitch_more_speed: 0.5  # 俯时yaw速度
    max_scan_period: 3     # 扫描周期数
    
    # 平滑参数
    yaw_smooth_factor: 0.8      # yaw角度平滑系数
    pitch_smooth_factor: 0.8    # pitch角度平滑系数
    smooth_window_size: 5       # 平滑窗口大小
    state_switch_threshold: 0.5 # 状态切换阈值,障碍物合并大小
    
    # 自瞄和装甲板参数
    armor_id: [1, 2, 3, 4, 5, 6, 7, 8, 9] # 期望的装甲板ID
    max_distance: 8.0       # 最大检测距离
    tracking_timeout: 1.5   # 自瞄超时时间（秒）
    transition_delay: 1.0   # 状态切换延迟时间（秒）
    enemy_distance: 3.0     # 敌人距离期待
